docs/topics/proc_starting-your-spring-boot-application-locally-in-debugging-mode.adoc

```diff
@@ -0,0 +1,28 @@
+[#starting-your-spring-boot-application-locally-in-debugging-mode]
+= Starting Your {SpringBoot} Application Locally in Debugging Mode
+
+One of the ways of debugging a Maven-based project is manually launching the application while specifying a debugging port, and subsequently connecting a remote debugger to that port.
+This method is applicable at least to the following deployments of the application:
+
+* When launching the application manually using the `mvn spring-boot:run` goal.
+* When using the Arquillian adapter for {runtime}.
```

Comment:
There's no Arquillian adapter for Spring Boot. (At least I'm not aware of one. There is probably a way how to test Spring Boot applications using Arquillian, but I don't really know anything about that. Should be discussed with the Spring Boot team.)